images/foreman/Containerfile

```
@@ -14,7 +16,7 @@ ENV RAILS_LOG_TO_STDOUT=true
 CMD /usr/share/foreman/bin/rails db:migrate && /usr/share/foreman/bin/rails db:seed && /usr/share/foreman/bin/rails server --environment production
 EXPOSE 3000/tcp
 
-ARG FOREMAN_PLUGINS="foreman-tasks foreman_remote_execution katello"
+ARG FOREMAN_PLUGINS="foreman-tasks foreman_remote_execution katello foreman-plugin-google foreman-plugin-azure_rm foreman-plugin-ovirt foreman-plugin-kubevirt"
```

```suggestion
ARG FOREMAN_PLUGINS="foreman-tasks foreman_remote_execution katello foreman_google foreman_azure_rm foreman_ovirt foreman_kubevirt"
```

While Ewoud suggested to use the `foreman-plugin-NAME` alias above, here we need the GEM notation (for now, at laest, as we need to know the gem name)